central/telemetry/centralclient/interceptors.go

```diff
@@ -0,0 +1,107 @@
+package centralclient
+
+import (
+	"strings"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+var (
+	trackedPaths set.FrozenSet[string]
+	ignoredPaths = []string{"/v1/ping", "/v1.PingService/Ping", "/v1/metadata", "/static/"}
+
+	uninitializedClusters = set.NewSet[string]()
+	mux                   = &sync.Mutex{}
+
+	interceptors = map[string][]phonehome.Interceptor{
+		"API Call":            {apiCall},
```

Comment:
I assume the plan is to start with a generic API capture and then replace them more customised events down the road?

Reply:
We actually start with 3 other events (sensor cluster registration/initialization and roxctl), while the "API Call" is there as a generic configurable "backdoor" for tracking specific calls without recompiling.